### PR TITLE
fix: typo on drawerPosition default props

### PR DESCRIPTION
**Pull request by diegolmello**
Fix a minor typo from `drawerPostion` to `drawerPosition` :)

Fix https://github.com/react-navigation/react-navigation/issues/8358

### CODE REVIEW

**Comment by tarbundiyahitesh21** [comment]
> Fix a minor typo from `drawerPostion` to `drawerPosition` :)

thanks, now it's working

File: packages/drawer/src/views/DrawerView.tsx [hunk 1]
```diff
@@ -238,7 +238,6 @@ export default function DrawerView({
               renderDrawerContent={renderNavigationView}
               renderSceneContent={renderContent}
               keyboardDismissMode={keyboardDismissMode}
-              drawerPostion={drawerPosition}
```
Comment (satya164) [inline packages/drawer/src/views/DrawerView.tsx:241]: Why did you remove it from here?

Comment (diegolmello) [reply packages/drawer/src/views/DrawerView.tsx:241]: @satya164 Because it's also a typo and it's duplicated.
[Here (correct)](https://github.com/react-navigation/react-navigation/blob/master/packages/drawer/src/views/DrawerView.tsx#L211) and [here (incorrect and removed)](https://github.com/react-navigation/react-navigation/blob/master/packages/drawer/src/views/DrawerView.tsx#L241)
